src/pages/beta/teachers-2023/early-release-units.tsx

```diff
@@ -3,11 +3,89 @@ import { NextPage } from "next";
 import { BETA_SEO_PROPS } from "@/browser-lib/seo/Seo";
 import AppLayout from "@/components/AppLayout/AppLayout";
 import EarlyReleaseUnitsHeader from "@/components/EarlyReleaseUnitsHeader";
+import EarlyReleaseExemplarUnits from "@/components/EarlyReleaseExemplarUnits";
+import earlyReleaseExemplarUnitsFixture from "@/node-lib/curriculum-api/fixtures/earlyReleaseExemplarUnits.fixture";
+import { Heading, Hr, P } from "@/components/Typography";
+import Box from "@/components/Box";
+import BrushBorders from "@/components/SpriteSheet/BrushSvgs/BrushBorders/BrushBorders";
+import Grid, { GridArea } from "@/components/Grid";
+import Card from "@/components/Card";
+import Flex from "@/components/Flex";
+import MaxWidth from "@/components/MaxWidth/MaxWidth";
+import AnchorTarget from "@/components/AnchorTarget";
 
 const EarlyReleaseUnits: NextPage = () => {
+  const primary = earlyReleaseExemplarUnitsFixture().primary;
+  const secondary = earlyReleaseExemplarUnitsFixture().secondary;
   return (
     <AppLayout seoProps={BETA_SEO_PROPS}>
       <EarlyReleaseUnitsHeader />
+      <MaxWidth>
+        <Flex
+          $justifyContent={"centers"}
```

Comment:
playing a fun new starter game of "is this a typo, or is this a new specific syntax I'm unfamiliar with?" 😬

Reply:
Ha! yeah a typo, good spot!  The question is why didn't he type check complain about this